presto-main/src/main/java/com/facebook/presto/sql/analyzer/MaterializedViewQueryOptimizer.java

```diff
@@ -83,6 +83,12 @@
 {
     private static final Logger logger = Logger.get(MaterializedViewQueryOptimizer.class);
 
+    private static final QualifiedName MIN = QualifiedName.of("MIN");
+    private static final QualifiedName MAX = QualifiedName.of("MAX");
+    private static final QualifiedName SUM = QualifiedName.of("SUM");
+    private static final QualifiedName COUNT = QualifiedName.of("COUNT");
+    private static final Set<QualifiedName> SUPPORTED_FUNCTION_CALLS = ImmutableSet.of(MIN, MAX, SUM, COUNT);
+
     private final Metadata metadata;
```

Comment:
Look at: TestAggregationOperator at the top you will see the way the use function manager. You can do that here (or better yet move it to some utility class)